src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.scss

```diff
@@ -52,16 +54,17 @@
   grid-column: 1/2;
   text-align: right;
   padding-right: 1em;
+  color: $dark-blue;
```

Comment:
<img src="https://user-images.githubusercontent.com/6834224/92145554-4dc70f00-ee10-11ea-9bb0-8bb965e4a834.png" width="150" />

revert?

Reply:
Waiting for confirmation from Andrea